main.js

```diff
@@ -0,0 +1,67 @@
+const Name = 'shaheer';
+const Age = 19;
+const Rating = 9.43;
+const Bio = true;
+const x = null;
+const y = undefined;
+let z;
+
+const todos = [
+	{
+		id: 1,
+		text='Take out Dog'
+		iscompleted: true,
+	},
+	{
+		id: 2,
+		text='Wake up at 6'
+		iscompleted: true
+	},
+	{
+		id: 3,
+		text='Go to Gym'
+		iscompleted: true
+	},
+	{
+		id: 4
+		text= 'Attend meeting'
+		completed: true
+	},
+	{
+		id: 5
+		text= 'Get Food'
+		completed: false
+	},
+]
+
+for (let i = 0; i <= 10; i++) {
+	console.log('for loop number: ${i}');
+}
+
+const addnums = num1 => num1 + 5;
+console.log(addnumms(5));
+
+function person(FirstName, LastName, Dob) {
```

Comment:
convert the function parameters to camel case i.e `firstName`, `lastName`, `dob`